shell/common/logging.cc

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2021 Slack Technologies, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "shell/common/logging.h"
+
+#include <string>
+
+#include "base/base_switches.h"
+#include "base/command_line.h"
+#include "base/environment.h"
+#include "base/files/file_path.h"
+#include "base/logging.h"
+#include "base/path_service.h"
+#include "base/strings/string_number_conversions.h"
+#include "chrome/common/chrome_paths.h"
+#include "content/public/common/content_switches.h"
+#include "shell/common/electron_paths.h"
+
+namespace logging {
+
+constexpr base::StringPiece kLogFileName("ELECTRON_LOG_FILE");
```

Comment:
Do we want to document this new env variable ?

Reply:
It was mentioned in `--log-file` docs, but i've added it to the environment variables doc too.